files/galaxy/config/tool_conf.xml

```diff
@@ -276,6 +276,8 @@
   </section>
   <section id="mimodd" name="MiModD">
   </section>
+  <section id="singlecell" name="Single-cell" >
```

Comment:
Should this be under RNA analysis?

Reply:
If there are enough tools, in a coherent suite/toolkit, then could make sense to have separate. If you'd rather they belong under RNA then could work too.

Reply:
Not under RNA section, sorry but under the entry ... mh, ok again. I think this should be under/in "Genomics Analysis" rather then "Genomics Toolkits"

Reply:
+1

Reply:
Oh I thought it was still Genomics Analysis. I missed the new label
Fix it in #37